rfcs/followers.md

```diff
@@ -0,0 +1,106 @@
+RFC: Follower cluster peers
+===========================
+
+
+1. Per https://github.com/ipfs/ipfs-cluster/pull/710/files we will introduce
+   RPC authorization policies:
+
+  * `raft` policy is still pretty much fully open (all endpoints that can
+  potentially be used by someone else are open to everyone
+  * `crdtsoft` allows trusted peers to perform sync/recover/block put etc on any other peer
+  * `crdtstrict` allows only read-only methods to trusted peers and almost nothing to the rest (connect swarms (?))
+
+Running with `crdtsoft` policy where all peers trust all the other peers
+should be equivalent to running with `raft` policy (right?).
+
+Regarding the configuration:
+
+* `rpc_auth_policy` key will be introduced in the main `cluster` section
+* `trusted_peers` array for trusted peer IDs will be introduced in the main `cluster` section.
+
+This brings the question:
+
+##  Does it make sense to make `raft` policy selectable (with a different name) for "trusted" environments running with CRDTS (so that the user does not need to add/rm trusted peers when they join/leave the cluster?
```

Comment:
I mean, when you want everyone in the cluster to be able to commit pins and unpins.

Reply:
yes, essentially